google/cloud/spanner/samples/samples.cc

```diff
@@ -681,6 +681,7 @@ void CreateBackupCommand(std::vector<std::string> const& argv) {
   CreateBackup(std::move(client), argv[0], argv[1], argv[2], argv[3]);
 }
 
+//! [restore-database] [START spanner_restore_backup]
```

Comment:
We need to remove the `[restore-database]` doxygen tag from this line or the next line.  If we eventually want to remove the `spanner_restore_database` region tag I would recommend moving the doxygen tag.  If we want to keep both region tags, then maybe removing `[restore-database]` from the new lines is better.  More on this can be found at go/cloud-cxx:snippets-dd

Reply:
This is one of multiple PRs spanning several repositories; I can't be certain how quickly those will get turned around (I'd rather not move the tag for now). 

I'm can remove the tag from the new line(s), and when I do a 2nd PR pass to remove the "old" `spanner_restore_database` I will move the doxygen tag at that time. SGTY?

Reply:
> I'm can remove the tag from the new line(s), and when I do a 2nd PR pass to remove the "old" spanner_restore_database I will move the doxygen tag at that time. SGTY?

Yup.